packages/playwright/src/reporters/base.ts

```diff
@@ -448,15 +448,18 @@ export function formatError(error: TestError, highlightCode: boolean): ErrorDeta
     tokens.push(snippet);
   }
 
-  if (parsedStack && parsedStack.stackLines.length) {
-    tokens.push('');
```

Comment:
Make exceptions more concise and it removes the spacing between message and stack (helps also when calling it recursively instead of tracking if its the first level)